source/val/Function.h

```diff
@@ -262,6 +265,12 @@ class Function {
   std::unordered_map<const BasicBlock*, std::vector<BasicBlock*>>
       augmented_predecessors_map_;
 
+  // Maps a structured loop header to its CFG successors and also its
+  // continue target if that continue target is not the loop header
+  // itself. This might have duplicates.
+  std::unordered_map<const BasicBlock*, std::vector<BasicBlock*>>
```

Comment:
This variable seems to be more aligned with a construct object then a function. Hmm, that would make things tricky. This is fine for now.